app/stylesheets/config/type.scss

```diff
@@ -1,3 +1,11 @@
+@import url(https://fonts.googleapis.com/css?family=Open+Sans:400,700,800,300,600);
+
+$tab-text: 12px;
+
 @mixin no-margin() {
   margin: 0;
 }
+
+@mixin link-transition($link-duration, $link-easing) {
+  transition: $link-duration, $link-easing;
```

Comment:
I think this needs `color` or `all` before `$link-duration`, if we keep this mixin (see comment in global.scss).